### ISSUE TITLE
How to update a deck by importing a file?

### ISSUE THREAD

**Issue by ermats** [tema6120/ForgetMeNot#77]
I prefer writing the Q&As on my computer and then import it into the app, but since over the course over a semester more and more questions come up, I need to update my decks. Is there a way to do this?

**Comment by tema6120** [comment]
On import screen it is possible to add only new cards from file to existing deck.

https://user-images.githubusercontent.com/34657415/148683744-7f03f183-0987-4168-810c-28ddb3170126.mp4

**Comment by tlfhnw** [comment]
Oh that's nice. Any way to also update existing questions/answers?

**Comment by SeerLite** [comment]
Yeah, is there a way to replace cards with matching questions? (While keeping learned progress of that card)

**Comment by maltfield** [comment]
See also:

 * https://github.com/tema6120/ForgetMeNot/issues/72
